internal/store/installation.go

```diff
@@ -144,6 +144,25 @@ func (sqlStore *SQLStore) GetInstallations(filter *model.InstallationFilter, inc
 	return installations, nil
 }
 
+// GetInstallationsCount returns the number of installations filtered by the deletedat
+// field
+func (sqlStore *SQLStore) GetInstallationsCount(includeDeleted bool) (int, error) {
+	builder := sq.Select("COUNT(*) as InstallationsNumber").From("Installation")
```

Comment:
nit: maybe this instead?
```suggestion
	builder := sq.Select("COUNT(*) as InstallationsCount").From("Installation")
```